hibernate-core/src/main/java/org/hibernate/bytecode/enhance/internal/javassist/EnhancerImpl.java

```diff
@@ -114,6 +116,9 @@ public ClassLoader getClassLoader() {
 	protected CtClass loadCtClassFromClass(Class<?> aClass) {
 		String resourceName = aClass.getName().replace( '.', '/' ) + ".class";
 		InputStream resourceAsStream = aClass.getClassLoader().getResourceAsStream( resourceName );
+		if ( resourceAsStream == null ) {
+			throw new UncheckedIOException( new FileNotFoundException ( "Not found: " + aClass.getClassLoader().getResource( resourceName ) ) );
```

Comment:
this will render as: "Not found: null" since `aClass.getClassLoader().getResource( resourceName ) )` is null inside the if branch.

Better to change it to:

throw new UncheckedIOException( new FileNotFoundException ( "Not found: " + resourceName ) );